apps/mobile/convex/confect/http-api.ts

```
@@ -0,0 +1,150 @@
+        return Option.none();
+      })
+    )
+);
+
+const MessagesApiGroupLive = HttpApiBuilder.group(OpenAgentsApi, "messages", (handlers) =>
+  handlers
+    .handle("getMessages", () =>
+      Effect.gen(function* () {
+        // TODO: Fix HTTP API integration with Confect functions
+        return [];
+      })
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)